did-tezos/src/lib.rs

```diff
@@ -90,18 +99,19 @@ impl DIDResolver for DIDTz {
             }
         };
 
-        let prefix = match address.get(0..3) {
-            Some(prefix) => prefix,
-            None => {
-                return (
-                    ResolutionMetadata::from_error(ERROR_INVALID_DID),
-                    None,
-                    None,
-                )
+        let prefix: Prefix = match address.get(0..3) {
+            Some(prefix) => {
+                match serde_json::from_str(&format!("\"{}\"", prefix.to_ascii_lowercase())) {
```

Comment:
Would implementing FromStr on Prefix be preferable? Aren't the prefixes case-sensitive?

Reply:
I don't have a strong preference, implemented in f6c44c9 (and 0072f84 for the case sensitiveness)